src/components/main/AddFolderButton.js

```diff
@@ -0,0 +1,24 @@
+import styles from "./AddFolder.module.css";
+
+function AddFolderButton() {
```

Comment:
```suggestion
function AddFolderButtons() {
```
버튼이 두 개 이상 `React.Fragment`로 감싸져있으니 복수형으로 짓는게 나을 것 같아요 !